generate-fit-profile/Cargo.toml

```diff
@@ -11,4 +11,6 @@ keywords = ["ant", "fit", "garmin"]
 
 [dependencies]
 calamine = "0.22"
+proc-macro2 = "1.0.78"
+quote = "1.0.35"
```

Comment:
Do we need to pin to patch numbers here? Unless it's recommended for those crates I tend to run with just major/minor which seems to give a nice balance of stability without updating dep versions as often.

Reply:
Ah got it, thanks. Again lots of time in Python/Java land lately and everyone does their reqs slightly different.